server/socketCommands.ts

```diff
@@ -2,12 +2,17 @@ import { Server, Socket } from 'socket.io'
 
 declare module 'socket.io' {
   interface Socket {
-      username: string
+      username: string,
+      status: boolean
```

Comment:
Can make this type 'read' | 'not ready' or any other string that you think makes more sense. This makes it that typescript will only allow the specific strings 'ready' and 'not ready'.